sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -809,6 +809,12 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val SHUFFLE_UCX_USE_SOCKADDR = conf("spark.rapids.shuffle.ucx.useSockaddr")
+    .doc("When set to true, start listener and exchange socket address. This allows better detecting " +
```

Comment:
This should include info on what the connectivity should be between peers. That we are exchanging socket address, seems like an implementation detail.

Can you describe the requirement here for `UcpListener` in the case where:

1. We have 1 NIC specified via UCX_NET_DEVICES
2. We have multiple NICs.

Can the NIC used for the listener be some third (non MLNX NIC for example)?

I am not asking for the above to be in the doc, but it may help us write something useful.

Reply:
So yes, if listener is bind to non mlx NIC then tcp_sockcm would be used to do wireup. If it's on mlx NIC then rdma_cm. So I am thinking to start listener always except of Management service. WDYT?